tilelang/contrib/cutedsl/cpasync.py

```
@@ -213,3 +289,49 @@ def prefetch_tma_descriptor(tma_desc, *, loc=None, ip=None) -> None:
     else:
         tma_desc_ptr = tma_desc
     nvvm.prefetch_tensormap(tma_desc_ptr.llvm_ptr, loc=loc, ip=ip)
+
+
+# ---------------------------------------------------------------------------
+# Mbarrier operations (merged from mbar.py)
+# ---------------------------------------------------------------------------
+
+from cutlass._mlir.dialects import llvm  # noqa: F811
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unused: `F811`)

Remove unused `noqa` directive

(RUF100)